workers/unity/Packages/com.improbable.gdk.core/Config/RuntimeConfig.cs

```diff
@@ -18,12 +18,14 @@ public static class RuntimeConfigDefaults
     /// </summary>
     public static class RuntimeConfigNames
     {
+        public const string SteamDeploymentTag = "steamDeploymentTag";
```

Comment:
if they are both only used for steam, I would move them together and not have one field at the top and the other one at the bottom of the class :)

Reply:
Thanks for pointing out. Fixed!